toolkit/tools/imagegen/configuration/systemconfig.go

```diff
@@ -154,6 +161,14 @@ func (s *SystemConfig) IsValid() (err error) {
 	}
 
 	//Validate PostInstallScripts
+
+	// Validate Networks
+	for _, network := range s.Networks {
+		if err = network.IsValid(); err != nil {
+			return fmt.Errorf("invalid [Network]: %w", err)
```

Comment:
Nit: might be also nice to say which network config (index + 1) is the erroneous one.

Reply:
Fixed